common/metrics/metric_defs.go

```diff
@@ -768,29 +768,45 @@ var (
-	QueueReaderCountHistogram                            = NewDimensionlessHistogramDef("queue_reader_count")
-	QueueSliceCountHistogram                             = NewDimensionlessHistogramDef("queue_slice_count")
-	QueueActionCounter                                   = NewCounterDef("queue_actions")
-	QueueActionFailures                                  = NewCounterDef("queue_action_errors")
-	ActivityE2ELatency                                   = NewTimerDef("activity_end_to_end_latency")
-	AckLevelUpdateCounter                                = NewCounterDef("ack_level_update")
-	AckLevelUpdateFailedCounter                          = NewCounterDef("ack_level_update_failed")
-	CommandTypeScheduleActivityCounter                   = NewCounterDef("schedule_activity_command")
-	CommandTypeCompleteWorkflowCounter                   = NewCounterDef("complete_workflow_command")
-	CommandTypeFailWorkflowCounter                       = NewCounterDef("fail_workflow_command")
-	CommandTypeCancelWorkflowCounter                     = NewCounterDef("cancel_workflow_command")
-	CommandTypeStartTimerCounter                         = NewCounterDef("start_timer_command")
-	CommandTypeCancelActivityCounter                     = NewCounterDef("cancel_activity_command")
-	CommandTypeCancelTimerCounter                        = NewCounterDef("cancel_timer_command")
-	CommandTypeRecordMarkerCounter                       = NewCounterDef("record_marker_command")
-	CommandTypeCancelExternalWorkflowCounter             = NewCounterDef("cancel_external_workflow_command")
-	CommandTypeContinueAsNewCounter                      = NewCounterDef("continue_as_new_command")
-	CommandTypeSignalExternalWorkflowCounter             = NewCounterDef("signal_external_workflow_command")
-	CommandTypeUpsertWorkflowSearchAttributesCounter     = NewCounterDef("upsert_workflow_search_attributes_command")
-	CommandTypeModifyWorkflowPropertiesCounter           = NewCounterDef("modify_workflow_properties_command")
-	CommandTypeChildWorkflowCounter                      = NewCounterDef("child_workflow_command")
+	TaskSchedulerThrottled      = NewCounterDef("task_scheduler_throttled")
+	QueueScheduleLatency        = NewTimerDef("queue_latency_schedule") // latency for scheduling 100 tasks in one task channel
+	QueueReaderCountHistogram   = NewDimensionlessHistogramDef("queue_reader_count")
+	QueueSliceCountHistogram    = NewDimensionlessHistogramDef("queue_slice_count")
+	QueueActionCounter          = NewCounterDef("queue_actions")
+	QueueActionFailures         = NewCounterDef("queue_action_errors")
+	ActivityE2ELatency          = NewTimerDef("activity_end_to_end_latency")
+	AckLevelUpdateCounter       = NewCounterDef("ack_level_update")
+	AckLevelUpdateFailedCounter = NewCounterDef("ack_level_update_failed")
+	CommandCounter              = NewCounterDef("command")
```

Comment:
This is the new metric. If you have a more specific name in mind I could use, let me know!